docs/UserGuide.md

```diff
@@ -144,15 +144,15 @@ Examples:
 
 Displays applicants in Intern Watcher that match one or more given fields.
 
-Format: `filter [g/GRADE] [i/INSTITUTION]…+ [c/COURSE]…+ [y/GRADUATION_YEAR_MONTH] [j/JOB]…+ [a/STATUS]…+ [s/SKILL]…+`
+Format: `filter [g/GRADE] [i/INSTITUTION]…+ [c/COURSE]…+ [y/GRADUATION_YEAR_MONTH] [j/JOB]…+ [a/APPLICATION_STATUS]…+ [s/SKILL]…+`
```

Comment:
My PR already includes changes for this in the UG. Might have merge conflicts later.